src/software/ai/hl/stp/stp_refbox_game_state_play_selection_test.cpp

```diff
@@ -225,23 +222,55 @@ class STPRefboxGameStatePlaySelectionTest
 TEST_P(STPRefboxGameStatePlaySelectionTest,
        test_play_selection_for_all_refbox_game_states)
 {
-    // TODO (Issue #1330): replace the ball with real parameterized values
-    world.updateRefboxGameState(GetParam());
-    world.updateGameStateBall(Ball(Point(), Vector(), Timestamp::fromSeconds(0)));
+    RefboxGameState game_state = std::get<0>(GetParam());
+    Ball ball                  = std::get<1>(GetParam());
 
+    world.updateRefboxGameState(game_state);
+    world.updateGameStateBall(ball);
     try
     {
         auto play_ptr = stp.calculateNewPlay(world);
     }
     catch (...)
     {
-        FAIL() << "No play for game state " << GetParam();
+        FAIL() << "No play for game state " << std::get<0>(GetParam());
     }
 }
 
-auto all_refbox_game_states = ::TestUtil::getAllRefboxGameStates();
+// TODO (Issue #1665): Include `BALL_PLACEMENT_US` and `BALL_PLACEMENT_THEM` when ball
+// placement states have plays
 
-// TODO (Issue #1330): Reenable these tests
-// INSTANTIATE_TEST_CASE_P(AllRefboxGameStates, STPRefboxGameStatePlaySelectionTest,
-//                        ::testing::ValuesIn(all_refbox_game_states.begin(),
-//                                            all_refbox_game_states.end()));
+// NORMAL_START is omitted since there is no preceding PREPARE state
+INSTANTIATE_TEST_CASE_P(
+    AllRefboxGameStates, STPRefboxGameStatePlaySelectionTest,
+    ::testing::Values(
+        std::make_tuple(RefboxGameState::HALT,
+                        Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0))),
```

Comment:
what's the point of parameterizing the ball? 

I just ran these tests locally, and it seems like any Position + Initial velocity doesn't effect the `calculateNewPlay` function?

I can set all of the parameters to `Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0)))` and the test still passes

Reply:
There was a TODO as follows:
`// TODO (Issue #1330): replace the ball with real parameterized values`

Previously, the parameters were all set to the default `Ball`.

Reply:
Right, but just curious what these parameterized values are supposed to test? The test still passes if I just replace all of the positions with `Ball(Point(0, 0), Vector(), Timestamp::fromSeconds(0)))`

Reply:
@jordanzliu Can you elaborate on what that TODO was intended for?

Reply:
Looks like plays based on RefboxGameState will be calculated regardless of the position of the ball. 

I.e if we receive `KICKOFF_US` we don't really care if the ball is in the centre, we still obey the rules.

So I think you can safely un-parameterize the ball and just leave it default for these tests.

Reply:
Ball has been un-parametrized.